tests/integration/utils/dbus-pulse-volume.cpp

```diff
@@ -140,11 +143,7 @@ bool DBusPulseVolume::setVolume(QString const & role, double volume)
         {
             QDBusVariant dbusVar(QVariant::fromValue(volume));
             QDBusReply<void> set_vol = accounts_interface_->call(QLatin1String("Set"),
-#ifdef HAS_LOMIRI_ACCTSERVICE_SOUND_SETTINGS
                                             QVariant::fromValue(QString("com.lomiri.AccountsService.Sound")),
-#else
-                                            QVariant::fromValue(QString("org.ayatana.AccountsService.Sound")),
-#endif
```

Comment:
address differently

Reply:
We need to do a HAS_LOMIRI_SOUND_SCHEMAS here and have an alternative plan, if HAS_LOMIRI_SOUND_SCHEMAS is false.